commit 2ba5b48938d752d09d65d1a01c8ff0d2228c5ab5
Author: Sascha Hauer <s.hauer@pengutronix.de>
Date:   Wed Nov 8 07:05:06 2023 +0100

    ubifs: describe function parameters
    
    With 16a26b20d2afd ("ubifs: authentication: Add hashes to index nodes")
    insert_node() and insert_dent() got a new function parameter 'hash'. Add
    a description for this new parameter.
    
    Signed-off-by: Sascha Hauer <s.hauer@pengutronix.de>
    Reported-by: kernel test robot <lkp@intel.com>
    Closes: https://lore.kernel.org/oe-kbuild-all/202311051618.D7YUE1Rr-lkp@intel.com/
    Reviewed-by: Zhihao Cheng <chengzhihao1@huawei.com>
    Signed-off-by: Richard Weinberger <richard@nod.at>

diff --git a/fs/ubifs/replay.c b/fs/ubifs/replay.c
index c59d47fe7939..17da28d6247a 100644
--- a/fs/ubifs/replay.c
+++ b/fs/ubifs/replay.c
@@ -365,6 +365,7 @@ static void destroy_replay_list(struct ubifs_info *c)
  * @lnum: node logical eraseblock number
  * @offs: node offset
  * @len: node length
+ * @hash: node hash
  * @key: node key
  * @sqnum: sequence number
  * @deletion: non-zero if this is a deletion
@@ -417,6 +418,7 @@ static int insert_node(struct ubifs_info *c, int lnum, int offs, int len,
  * @lnum: node logical eraseblock number
  * @offs: node offset
  * @len: node length
+ * @hash: node hash
  * @key: node key
  * @name: directory entry name
  * @nlen: directory entry name length